File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.Collections;
import java.util.Date;
import java.util.List;

import no.nav.domain.pensjon.regler.beregning2011.SisteBeregning;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

public class VilkarsprovRequest extends ServiceRequest{
	private  Kravhode kravhode;
    private SisteBeregning sisteBeregning;
	private Date fom;
	private Date tom;
	private List<VilkarsVedtak>  vilkarsvedtakliste;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.SisteBeregning
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.time.LocalDate

class VilkarsprovRequest : ServiceRequest {
    var kravhode: Kravhode? = null
    var sisteBeregning: SisteBeregning? = null
    var fomLd: LocalDate? = null
    var tomLd: LocalDate? = null
    var vilkarsvedtakliste: List<VilkarsVedtak> = mutableListOf()

    constructor()


    constructor(kravhode: Kravhode?, sisteBeregning: SisteBeregning?, fom: LocalDate?, tom: LocalDate?) {
        this.kravhode = kravhode
        this.sisteBeregning = sisteBeregning
        this.fomLd = fom
        this.tomLd = tom
        this.vilkarsvedtakliste = mutableListOf()
    }
}